docs/_getting-started-macos-android.md

```diff
@@ -24,11 +24,22 @@ We recommend installing the OpenJDK distribution called Azul **Zulu** using [Hom
 ```shell
 brew install --cask zulu@17
 
-# Get path to where cask was installed to double-click installer
+# Get path to where cask was installed to find the JDK installer
 brew info --cask zulu@17
+
+# ==> zulu@17: <version number>
+# https://www.azul.com/downloads/
+# Installed
+# /opt/homebrew/Caskroom/zulu@17/<version number> (185.8MB) (note that the path is /usr/local/Caskroom on non-Apple Silicon Macs)
+# Installed using the formulae.brew.sh API on 2024-06-06 at 10:00:00
+
+# Navigate to the folder
+finder /opt/homebrew/Caskroom/zulu@17/<version number> # or /usr/local/Caskroom/zulu@17/<version number>
```

Comment:
`finder` did not work for me but `open` did
<img width="466" alt="Screenshot 2024-10-27 at 2 36 18 PM" src="https://github.com/user-attachments/assets/059bd26f-5e67-4987-af08-80e5c5dc6593">